Trappist/src/Promact.Trappist.Core/Controllers/ProfileDetailsController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.Repository.ProfileDetails;
+using Promact.Trappist.Web.Models;
+using System.Threading.Tasks;
+
+namespace Promact.Trappist.Core.Controllers
+{
+  [Route("api/Profile")]
+  public class ProfileDetailsController : Controller
+  {
+    private readonly IProfileRepository _profileRepository;
+    private UserManager<ApplicationUser> _userManager;
+    private readonly SignInManager<ApplicationUser> _signInManager;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the '_signInManager' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)